src/iroh.udl

```diff
@@ -98,11 +139,17 @@ dictionary CounterStats {
 
 interface LiveEvent {
   LiveEventType type();
+  [Throws=IrohError]
```

Comment:
Why? the point of these was to panic if misused, eg should have checked the type before and then converted

Reply:
was just making all the APIs for these kinds of enums the same. If we would rather go with panics, I'm all for it!